packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -261,12 +266,23 @@ export enum OutgoingPaymentState {
 }
 
 export enum OutgoingPaymentDepositType {
```

Comment:
Technically, neither the cancelled nor the funded events are "deposit" or "funded" type. We can either check whether the `OutgoingPaymentWithdrawType` or `OutgoingPaymentDepositType` are used anywhere and if they are, just make a third category for the cancelled or funded.